crates/turbopack-node/src/pool.rs

```diff
@@ -363,11 +371,17 @@ impl NodeJsOperation {
 
 impl Drop for NodeJsOperation {
     fn drop(&mut self) {
-        if let Some(process) = self.process.take() {
-            self.processes
-                .lock()
-                .unwrap()
-                .push(NodeJsPoolProcess::Running(process));
+        if let Some(mut process) = self.process.take() {
+            if self.in_bad_state {
+                if let Some(mut child) = process.child.take() {
+                    let _ = child.start_kill();
```

Comment:
Why not `child.kill()`?

Reply:
I copied that from our kill_and_wait message. I think @alexkirsz wrote that

Reply:
oh I see, this is a tokio process, we can't synchronously kill that